R/abundance_normalization.R

```diff
@@ -187,7 +187,7 @@ NormalizeMPX.MPXAssay <- function(
   ...
 ) {
   # If object has been merged before, join layers
-  if (inherits(object, "CellGraphAssay5")) {
+  if (inherits(object, c("CellGraphAssay5", "Assay5"))) {
```

Comment:
Should this be "Assay" as well, or not?

Reply:
As far as I understand the JoinLayers line below is only applicable for a v5 object. From v5 on Seurat stores data in layers. This can be multiple layers for merged objects, and these should be joined before normalization otherwise it only uses applies the normalization on the first layer.
If not v5, this joining can be skipped.